templates/graph/index.html

```diff
@@ -305,6 +306,7 @@ <h4 class="zero-margin d-inline-block">
 
     <!-- JSON support for Graph Page -->
     <script type="text/javascript" src="{% static 'js/gs_graph_stylesheet.js' %}"></script>
+    <script type="text/javascript" src="/static/js/jspdf.min.js"></script>
```

Comment:
I would suggest installing **jspdf** as bower dependency and updating bower.json.
Then you can reference jspdf.min.js from `static/components/jspdf/dist`.
Refer to line 304 on how it is being done.

Also, use the templating language `{% static ........}` instead of `/static/js/....`